### PR TITLE
rewrote code

### PR DESCRIPTION
**Pull request by W-Schkulov**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -7,11 +7,12 @@ function App() {
     <>
       <h1>Theme Creator</h1>
 
-      {initialColors.map((color) => {
-        return <Color key={color.id} color={color} />;
-      })}
+      <div className="color-container">
```
Comment (F-Kirchhoff) [inline src/App.jsx:10]: since this is a list of elements, you should use ul / li elements instead.

File: src/App.jsx [hunk 2]
```diff
@@ -7,11 +7,12 @@ function App() {
...
+        {initialColors.map((color) => {
+          return <Color key={color.id} color={color} />;
```
Comment (F-Kirchhoff) [inline src/App.jsx:12]:
```suggestion
          return <li key={color.id}><Color  color={color} /></li>;
```

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Wladimir, please update your code to use ul /li elements. everything else is fine 👍

**Comment by E-LeeSchaffer** [comment]
looks good

**Pull request merged**